setup.cfg

```diff
@@ -47,3 +48,4 @@ exclude =
 # W503: old coding style (new PEP8 is enforced by W504)
 ignore = I801,RST304,N999,W503
 max-complexity = 10
+known-modules = websocket-client:[websocket]
```

Comment:
not sure why flake8-requirements isn't picking this up automatically?

Reply:
yeah i'm not sure if it's looking at the info in setup.py/setup.cfg or Pipfile/Pipfile.lock, but the websocket-client setup.py seems to declare that it owns the websocket module, so unsure why it's not picking it up.